gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapper.java

```diff
@@ -123,12 +123,10 @@ private Condition getCondition(RCMRMT030101UK04EhrExtract ehrExtract, Patient pa
         currentComposition.getParticipant2()
             .stream()
             .findFirst()
-            .ifPresent(participant2 -> condition.setAsserter(new Reference(participant2.getAgentRef().getId().getRoot())));
+            .ifPresent(participant2 -> condition.setAsserter(
+                new Reference("Practitioner/" + participant2.getAgentRef().getId().getRoot()))
```

Comment:
Could possibly use new IdType(ResourceType.Practitioner.name(), participant2.getAgentRef().getId().getRoot())
Rather than using this string? Just a suggestion though

Reply:
Done :)